src/redux/configureStore.js

```diff
@@ -1,9 +1,16 @@
-import { createStore, applyMiddleware } from 'redux';
+import { createStore, applyMiddleware, combineReducers } from 'redux';
 import thunk from 'redux-thunk';
 // import logger from 'redux-logger';
 import { composeWithDevTools } from 'redux-devtools-extension';
-import rocketReducer from './rockets/rockets';
+import rocketReducer, { initialState } from './rockets/rockets';
+import { missionsReducer, missionsInitialState } from './missions/missions';
 
-const store = createStore(rocketReducer, composeWithDevTools(applyMiddleware(thunk)));
+const rootReducer = combineReducers({
+  missions: missionsReducer,
+  rockets: rocketReducer
+});
+const store = createStore((state, action) => rootReducer(state, action),
+  { rockets: initialState, missions: missionsInitialState },
+  composeWithDevTools(applyMiddleware(thunk)));
```

Comment:
You can directly pass rootReducer as an argument in createStore() , since each of the reducers has access to the state, you don't need to do this explicitly.

Reply:
Which one do you think is easy? This one
const store = createStore(rootReducer, composeWithDevTools(applyMiddleware(thunk)))
or the one you wrote? What you are doing is actually duplicating what combineeReducer is supposed to do. So it is obsolete.

Reply:
when we code we should consider the case when we have more contributors with more reducers in this case we have all initialstates and reducers ranged in the same file configureStore

Reply:
I get your point. Thanks for clearifying this.